pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -70,6 +71,7 @@
 , withMachined ? true
 , withNetworkd ? true
 , withNss ? true
+, withOomd ? true
```

Comment:
is there anything in here that users need to configure? Can we test it being present in a VM test?

Reply:
It requires a service
example/systemd/system/systemd-oomd.service
and some dbus stuff. Right now its not enabled.
share/dbus-1/system.d/org.freedesktop.oom1.conf

Reply:
And it wants swap: 

```
sudo /nix/store/0y9z2b5c245y0gkj1cw0q212963421dy-systemd-247-rc2/lib/systemd/systemd-oomd
Requires swap to operate
```

Reply:
We can also disable it by default until someones wants to properly integrate it?

Reply:
yes, I think we should only enable things which we have integration tests for. We can keep the flag, so it's easier for people to enable it when they want to start working on it (like we did with homed)